endpoints/kubernetes/grpc-bookstore.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: esp-grpc-bookstore
+spec:
+  ports:
+  # Port that accepts JSON/HTTP requests over HTTP.
+  - port: 80
+    protocol: TCP
+    name: http
+  # Port that accepts gRPC and JSON/HTTP2 requests over HTTP.
+  - port: 9000
+    protocol: TCP
+    name: http2
+  selector:
+    app: esp-grpc-bookstore
+  type: LoadBalancer
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: esp-grpc-bookstore
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: esp-grpc-bookstore
+    spec:
+      containers:
+      - name: esp
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
+        args: [
+          "-p", "80",
+          "-P", "9000",
+          "-s", "SERVICE_NAME",
+          "-v", "SERVICE_VERSION",
```

Comment:
fyi, the public doc still says "SERVICE_VERSION": https://cloud-dot-devsite.googleplex.com/endpoints/docs/quickstart-container-engine

@jeffmendoza 
how would you recommend? we need to settled down one and be consistency